django/conf/locale/ms/formats.py

```diff
@@ -0,0 +1,36 @@
+# This file is distributed under the same license as the Django package.
+#
+# The *_FORMAT strings use the Django date format syntax,
+# see https://docs.djangoproject.com/en/dev/ref/templates/builtins/#date
+DATE_FORMAT = 'j M Y'                   # '25 Oct 2006'
+TIME_FORMAT = 'P'                       # '2:30 p.m.'
+DATETIME_FORMAT = 'j M Y, P'            # '25 Oct 2006, 2:30 p.m.'
+YEAR_MONTH_FORMAT = 'F Y'               # 'October 2006'
+MONTH_DAY_FORMAT = 'j F'                # '25 October'
+SHORT_DATE_FORMAT = 'd/m/Y'             # '25/10/2006'
+SHORT_DATETIME_FORMAT = 'd/m/Y P'       # '25/10/2006 2:30 p.m.'
+FIRST_DAY_OF_WEEK = 1                   # Monday
```

Comment:
Can you confirm that it's Monday, not Sunday?

Reply:
Confirmed that it's Monday (reference: [https://savvytime.com/week-number/malaysia/2021](https://savvytime.com/week-number/malaysia/2021))

Reply:
I found sources that claim that it's Sunday :thinking: :
- https://www.timeanddate.com/calendar/?year=2021&country=69
- https://en.wikipedia.org/wiki/Names_of_the_days_of_the_week#Days_numbered_from_Sunday

Reply:
@sdil Can you take a look? Thanks!

Reply:
I'm gonna have to agree with you. I checked locally-produced calendars and they begin with Sunday. 

- [https://www.tourism.gov.my/promotional/calendar-2021](https://www.tourism.gov.my/promotional/calendar-2021)
- [https://www.permohonan.my/kalendar-cuti-umum-sekolah-malaysia/](https://www.permohonan.my/kalendar-cuti-umum-sekolah-malaysia/)
- [https://www.onestopmalaysia.com/public-holidays-2021.html](https://www.onestopmalaysia.com/public-holidays-2021.html)

Let me make the change and commit.

Reply:
This is a bit complicated here in Malaysia. Some states declare Sunday as the start day of the week while others declare Monday as the start of the week.

You can refer to this table: https://en.wikipedia.org/wiki/Workweek_and_weekend#Around_the_world

Based on the calendars, I agree that the start day of the week is Sunday.

Reply:
@sdil Thanks for checking :+1:

Reply:
My pleasure :)